app/controllers/ingredients_controller.rb

```diff
@@ -0,0 +1,7 @@
+class IngredientsController < ApplicationController
+
+  def index
+    @ingredients = Ingredient.order(:name)
```

Comment:
Doing `.order` here is an AR method beyond simple CRUD functionality, so it's better off (per MVC convention) going in a class method in the Ingredient model and being called here!